tests/datasets/test_tau2020uas_mobile.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+
+from tests.test_utils import run_clip_tests
+
+from soundata import annotations
+from soundata.datasets import tau2020uas_mobile
+
+
+TEST_DATA_HOME = "tests/resources/sound_datasets/tau2020uas_mobile"
+
+
+def test_clip():
+    default_clipid = "airport-barcelona-0-0-a"
+    dataset = tau2020uas_mobile.Dataset(TEST_DATA_HOME)
+    clip = dataset.clip(default_clipid)
+
+    expected_attributes = {
+        "audio_path": (
+            "tests/resources/sound_datasets/tau2020uas_mobile/TAU-urban-acoustic-scenes-2020-mobile-development/audio/airport-barcelona-0-0-a.wav"
```

Comment:
Just to confirm, all test clips are <= 1s, yes?

Reply:
Yes!